### DISCUSSION TITLE
已解决：win10关于potplayer挂载阿里云盘报错webdav无法修改此文件夹

### DISCUSSION THREAD

**Discussion by xiuzhi233** [AlistGo/alist#5496] [Q&A]
windows 默认的 WebClient 服务仅支持 https 协议，而alist仅支持dav。
解决办法：找到注册表路径\HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\WebClient\Parameters下的BasicAuthLevel条目，数据值1修改为2；接着到服务里面重启WebClient，搞定！（参考：http://t.csdnimg.cn/Toak8）

**Comment by lanxiaoo** [comment]
按上述操作后还是提示同样的错误啊

**Reply by Tebio** [reply]
确实，按步骤操作还是显示无法在FTP/WebDAV/HTTP上修改该文件夹

**Reply by cuxt** [reply]
我尝试以下方法解决了问题 搜索internet选项，选择高级，勾选TSL1.1、1.2、1.3（原帖：https://blog.leonus.cn/2023/alist-potplayer.html）

**Comment by Deepseaon** [comment]
试了下还可以选择直接添加专辑，在添加的选项中填写从wabdav搜寻，这样也能直接连上

**Comment by GitCourser** [comment]
我遇到同样的问题, 上面两个方法都没用, 出现这个提示有好几种可能
第1个改WebClient权限的不管用, 因为这个服务根本不用启动就能访问webdav
第2个改tls需要你webdav是启用的https才相关
最后发现potplayer连webdav必须要密码,空密码的也会提示`无法在FTP/WebDav/HTTP上修改该文件夹`,设置个密码就行了

**Comment by FireFoxzzzz** [comment]
路径中去掉":port number"，尝试“ip/dav”，确认端口是否正确，用户名和密码是否正确

**Reply by zhongbuzhong** [reply]
解决了 感谢！

**Reply by FreeAlieX** [reply]
权限给够，端口号别弄错，ip/dav
就调通了，谢谢

**Reply by Karen12138** [reply]
这个是可以的

**Comment by BlingDan** [comment]
admin权限不够，在WebClient中勾选下
![image](https://github.com/user-attachments/assets/0fc51da5-9522-4555-996c-5d547b1d0706)
![image](https://github.com/user-attachments/assets/7e32b355-1991-4eaf-ac12-909e5b129263)

**Reply by FREDdjdjfkk** [reply]
感谢

**Reply by doukangtai** [reply]
感谢

**Reply by SunAoran02** [reply]
太权威了哥，搞一上午了，👍，感谢

**Reply by zhanghouyin** [reply]
我靠 太有用了，我之前也是这种情况，结果这次居然忘记了 唉

**Reply by FreeAlieX** [reply]
通了，谢谢
